web-app.yaml

```diff
@@ -49,6 +49,6 @@ spec:
     app: webapp
   ports:
     - protocol: TCP
-      port: 8081
+      port: 5000
```

Comment:
thank you but my web deployment has some error t keep asking the username password  but i am providing the same we created in initial ,also changes into simple id& pass but still cannot access can you suggest some solving methods